addons/mail/static/src/components/file_uploader/file_uploader_tests.js

```diff
@@ -87,6 +88,49 @@ QUnit.test('no conflicts between file uploaders', async function (assert) {
     );
 });
 
+QUnit.test('handles pre-upload removals of attachments', async function (assert) {
```

Comment:
replace generic word "handles" by a more descriptive phrase of what should actually happen (or not) when removing an attachment pre-load